addon/components/virtual-each.js

```diff
@@ -163,11 +164,13 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
     const sanitizedIndex = Math.min(startingIndex, maxVisibleItemTop);
     const sanitizedPadding = (startingPadding > maxPadding) ? maxPadding : startingPadding;
 
-    this.calculateVisibleItems(sanitizedIndex);
-    this.$().scrollTop(sanitizedPadding);
-  },
+    emberRun.scheduleOnce('afterRender', () => {
```

Comment:
We should cancel the scheduling on destroy just to be guard against this being dequeued when the component is destroyed || destroying